.github/workflows/check-for-changes.yml

```diff
@@ -1,4 +1,4 @@
-name: Check for Changes
+name: Check for Changes & Sync from Fork
```

Comment:
might want to rename this file as well, something like `sync-from-fork.yml`

Reply:
and `Sync from fork` would be enough as a name here

Reply:
Fixed